jib-core/src/main/java/com/google/cloud/tools/jib/image/json/ContainerConfigurationTemplate.java

```diff
@@ -152,6 +184,41 @@ public void setContainerCmd(@Nullable List<String> cmd) {
     config.Cmd = cmd;
   }
 
+  public void setContainerHealthTest(@Nullable List<String> test) {
+    if (config.Healthcheck == null) {
+      config.Healthcheck = new HealthCheckObjectTemplate();
```

Comment:
This looks like you can never go back to the `null` `config.Healthcheck` state. Might be fine though.

Reply:
Yeah I think this is fine given our use case for this class.